### ISSUE TITLE
508 [SCREENREADER]: Hide progress button icon from screen-readers

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#30724]
## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _First name only. Delete if not applicable._

## User Story or Problem Statement

As a screen-reader user, I don't want the `«` or `»` symbols read out with the "Back" or "Continue" text, respectively.

## Details

The `button-icon` span that wraps the double arrow symbols needs to be hidden from screen-readers. See screenshot

## Acceptance Criteria

- [ ] `button-icon` span is hidden from screen-readers

## Environment

All

## Steps to Recreate

Enable a screen-reader within any in-progress form and focus on the back or continue button

## Proposed Solution (if known)

```html
<span className="button-icon" aria-hidden="true">«</span>
<span className="button-icon" aria-hidden="true">»</span>
```

## Screenshots or Trace Logs

![Rendered back and continue buttons in the upper half with developer window in the bottom half showing HTML spans with only a button-icon class](https://user-images.githubusercontent.com/136959/135339124-bac4ccbb-db6e-4f2d-a42c-4550af7c2e58.png)

Screen reader _may_ ignore the symbol, but it is still visible

<img width="806" alt="Continue button with adjacent voice over window showing the double forward arrow icon plus continue text" src="https://user-images.githubusercontent.com/136959/135338905-299a0768-cbe8-441b-aaeb-e924c54dcc42.png">

**Comment by Mottie** [comment]
Fix merged into master. It should be available for testing on staging soon.

**Comment by Mottie** [comment]
Oops, I fixed the `ProgressButton` component that isn't used any more... the component-library update will fix this issue.
